modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -84,6 +84,7 @@ class ONNXImporter
                                     const opencv_onnx::GraphProto& graph_proto);
     Mat getBlob(const opencv_onnx::NodeProto& node_proto, int index);
     Mat getBlob(const std::string& input_name);
+    Mat getIntBlob(const opencv_onnx::NodeProto& node_proto, int index);
```

Comment:
Overload `getBlob` or update the signature? e.g. `Mat getBlob(const opencv_onnx::Nodeproto& node_proto, int index, int return_dtype)`.

Reply:
I think it will not look good, because the logic of the functions is different: getBlob should accept any type and return the raw blob, getIntBlob should accept only int64 or int32 types and return the blob converted to int32

Reply:
Ok.